themes/ddbasic/sass/components/node/campaign.scss

```diff
@@ -86,34 +89,37 @@
   // text only layout
   &.text {
     background: $color-primary;
-    color: $color-text-on-primary;
-
-  .ding-campaign-text {
-    @include span-columns(12);
-    padding: 40px 30px;
 
-    .ding-campaign-headline {
-      @include span-columns(8);
+    a {
+      color: $color-text-on-primary;
```

Comment:
Could this be moved to the general link styling for ` .ding-campaign a` to reduce nesting? It is also duplicated now between two campaign types.